packages/primitives/test/css.test.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import { css } from '@emotion/primitives'
+import { css } from '../src'
```

Comment:
This should be `@emotion/primitives`, all package names are aliased to their src.